hawc/tools/tables/sot.py

```diff
@@ -0,0 +1,174 @@
+from enum import Enum
+from typing import List, Optional
+
+import pandas as pd
+from pydantic import BaseModel, Field, conint
+
+from hawc.apps.animal.models import Endpoint
+from hawc.apps.materialized.models import FinalRiskOfBiasScore
+from hawc.apps.riskofbias.constants import SCORE_SHADES, SCORE_SYMBOLS
+
+from .base import BaseCell, BaseCellGroup, BaseTable
+from .generic import GenericCell
+from .parser import QuillParser, color_background, tag_wrapper
+
+
+class AttributeChoices(Enum):
+    RobScore = "rob_score"
+    StudyName = "study_name"
+    SpeciesStrainSex = "species_strain_sex"
+    Doses = "doses"
+    FreeHtml = "free_html"
+
+    def get_rob_score(self, selection: pd.DataFrame) -> BaseCell:
```

Comment:
it would be great to have a unit-test for each of these methods. I think we may have to replicate this on the frontend too